packages/db/prisma/schema.prisma

```diff
@@ -43,7 +43,8 @@ model Activity {
 }
 
 model Absence {
-  id          Int      @id @default(autoincrement())
   userId      Int
-  absenceDate DateTime @db.Date
+  absenceDate DateTime
```

Comment:
is there a reason why you removed the `@db.Date`?

Reply:
mb i thought it was the reason the check failed on one of the commits but that was just bc i didn't change the absence.ts file yet :o